CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 <!--
 Generated from:
 https://github.com/zopefoundation/meta/tree/master/config/pure-python
--->
+--> 
```

Comment:
It is interesting, that we add another whitespace here. Is that intended? There is now pre-commit whitespace check to correct it.